### PR TITLE
exercises for chapter 6, 2 and 3

### PR DESCRIPTION
**Pull request by mr-l-e-t**

### CODE REVIEW

File: chapter_06/lucas/ex01.pl [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+my %last_name = (  # a hash may be a lexical variable
+  fred   => 'flintstone',
+  barney => 'rubble',
+  wilma  => 'flintstone',
+);
+
+print "please nter your first name: ";
+chomp($first_name = <STDIN>);
+
+print "your last name is $last_name{$first_name} \n";
```
Comment (jhhmastercard) [inline chapter_06/lucas/ex01.pl:13]: Would be nice to check first_name exists.

File: chapter_06/lucas/ex03.pl [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/perl
+use utf8;
+
+foreach $k (sort(keys %ENV)) {
+	print "key: $k, \t\t\t value: $ENV{$k} \n";
+	print "---------------------------------------------------------------------------\n";
```
Comment (jhhmastercard) [inline chapter_06/lucas/ex03.pl:6]: This doesn't align the columns correctly

File: chapter_06/lucas/ex02.pl [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/perl
+use utf8;
+
+print "please enter a list of names. terminate line input with ctrl+z and enter: ";
```
Comment (jhhmastercard) [inline chapter_06/lucas/ex02.pl:4]: ctrl+d?

Comment (alexdeas) [reply chapter_06/lucas/ex02.pl:4]: Ctrl Z sends EOF on Windows.

File: chapter_06/lucas/ex01.pl [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+use utf8;
+
+my %last_name = (  # a hash may be a lexical variable
+  fred   => 'flintstone',
+  barney => 'rubble',
+  wilma  => 'flintstone',
+);
+
+print "please nter your first name: ";
```
Comment (jhhmastercard) [inline chapter_06/lucas/ex01.pl:10]: typo: nter -> enter

**Pull request merged**